model/gen-ai/spans.yaml

```diff
@@ -28,9 +28,10 @@ groups:
         requirement_level: recommended
       - ref: gen_ai.request.seed
         requirement_level:
-          conditionally_required: if appliable and if the request includes a seed
-      - ref: gen_ai.request.encoding_formats
-        requirement_level: recommended
```

Comment:
just checking that removing this ref was intentional?

Reply:
great catch, thanks!